src/main/java/org/folio/consortia/service/impl/SharingRoleCapabilitySetService.java

```diff
@@ -71,13 +80,13 @@ protected void validateSharingConfigRequestOrThrow(UUID roleId,
       throw new IllegalArgumentException("Payload must not be null");
     }
     if (!sharingRoleRepository.existsByRoleId(roleId)) {
-      throw new ResourceNotFoundException("roleId", String.valueOf(roleId));
+      throw new ResourceNotFoundException(ID_FIELD, String.valueOf(roleId));
```

Comment:
name a bit confuse. Your can rename to ROLE_ID as example, because when reading it sounds like this field is just "id"